src/scripts.js

```diff
@@ -8,42 +8,60 @@ import SleepRepo from "./SleepRepo";
 import Activity from "./Activity";
 import ActivityRepo from "./ActivityRepo";
 
-let activityData;
-let userData;
-let userRepo;
-let activityRepo;
-let activity;
-let user;
-const uniqueUserIndex = Math.floor(Math.random() * (50 - 1 + 1)) + 1;
-
-fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
-.then(response => response.json())
-.then(data => userData = data.userData)
-.then(() => {
-userRepo = new UserRepo(userData);
-user = new User(userData[uniqueUserIndex]);
-})
-.then(() => {activityFetch(),setTimeout(all, 100)})
+//Generate random user
```

Comment:
Another comment line we can probably remove

Reply:
Ok I've seen a couple now and I understand why its here.  We should all talk and see if this something we want to keep in our code base.

Reply:
@ckoga Yeah, it's stuff Pol and Ayla put in.  I can remove it if you'd prefer.